### PR TITLE
Updated Vercel install command to use pnpm

### PR DESCRIPTION
**Pull request by zckly**
Previously, the screenshot said to use `npm install --prefix=../..` but this will throw an error in Vercel. After changing it to use `pnpm i -r`, the build step works properly!

### CODE REVIEW

**Review by juliusmarminge — COMMENTED** [review]
Would you mind changing the command to

`pnpm --filter \!expo install`

or even with the store

`pnpm --filter \!expo install --store=node_modules/.pnpm-store`

It's what I use on the deployment for this repo. It skips all the expo stuff which saves a couple seconds and cache size :)

**Comment by zckly** [comment]
> Would you mind changing the command to
> 
> `pnpm --filter \!expo install`
> 
> or even with the store
> 
> `pnpm --filter \!expo install --store=node_modules/.pnpm-store`
> 
> It's what I use on the deployment for this repo. It skips all the expo stuff which saves a couple seconds and cache size :)

Just updated! 

CleanShot 2022-11-15 at 11 34 16@2x

File: README.md [hunk 1]
```diff
@@ -96,7 +96,8 @@ _We do not recommend deploying a SQLite database on serverless environments sinc
 Let's deploy the Next.js application to [Vercel](https://vercel.com/). If you have ever deployed a Turborepo app there, the steps are quite straightforward. You can also read the [official Turborepo guide](https://vercel.com/docs/concepts/monorepos/turborepo) on deploying to Vercel.
 
 1. Create a new project on Vercel, select the `apps/nextjs` folder as the root directory and apply the following build settings:
-   <img width="907" alt="CleanShot 2022-09-03 at 22 51 25@2x" src="https://user-images.githubusercontent.com/51714798/188287309-e6ff4cb9-827a-4e50-83ed-e0953d7752f9.png">
+
+<img width="927" alt="CleanShot 2022-11-15 at 11 34 16@2x" src="https://user-images.githubusercontent.com/11340449/201974887-b6403a32-5570-4ce6-b146-c486c0dbd244.png">
```
Comment (juliusmarminge) [inline README.md:100]:
```suggestion
<img width="927" alt="Vercel deployment settings" src="https://user-images.githubusercontent.com/11340449/201974887-b6403a32-5570-4ce6-b146-c486c0dbd244.png">

> The install command filters out the expo package and saves a few second (and cache size) of dependency installation. The build command makes us build the application using Turbo.
```

**Review by juliusmarminge — APPROVED** [review]
Thanks 👍🏼

**Pull request merged**